client/app/components/ApplicationArea/ApplicationLayout/VersionInfo.jsx

```diff
@@ -14,8 +14,8 @@ export default function VersionInfo() {
         <div className="m-t-10">
           {/* eslint-disable react/jsx-no-target-blank */}
           <Link href="https://version.redash.io/" className="update-available" target="_blank" rel="noopener">
-            Update Available
-            <i className="fa fa-external-link m-l-5" />
+            Update Available <i className="fa fa-external-link m-l-5" aria-hidden="true" />
+            <span className="sr-only">&#40;External link&#41;</span>
```

Comment:
I found escaping more agreeable than interpolating, but it's more of a stylistic choice. I really like your suggested approach of embarking this logic inside `<Link>` and it's what most current standards are moving toward. Maybe it extends a bit too much the scope of this work. I'll follow up on it in an issue.

About "external link" vs "opens in a new tab". I might agree the second one is better, but at the same time the intention of a11y is to convey the same meaning the icon should have. Also, it's likely screen readers can interpret `target="_blank"` by themselves.

Reply:
> Maybe it extends a bit too much the scope of this work. I'll follow up on it in an issue.

Sure, it's a relatively small refactor, but it changes a component that is used along the app entirely, so whatever works best for you.

> About "external link" vs "opens in a new tab". I might agree the second one is better, but at the same time the intention of a11y is to convey the same meaning the icon should have. Also, it's likely screen readers can interpret target="_blank" by themselves.

🤔 My understanding was actually in the other direction, that we wanted to highlight the "open in new tab" behavior rather than the icon. Also the `fa-external-link` icon is actually used as "opens in new tab", despite of its name. Re. the screen readers, I'm not sure, when I checked from a quick google search the recommendations were on the side that we should clarify the new window/tab behavior disregarding the `target="_blank"`